typegate/tests/runtimes/prisma/full_prisma_mapping_test.ts

```diff
@@ -41,7 +41,7 @@ Meta.test("prisma full mapping", async (t) => {
                 }
               }
               extended_profile: {
-                create: { id: 10111, bio: "Some bio 1" }
+                create: { bio: "Some bio 1" }
```

Comment:
` config={"id": ["user"]}` sets it auto by default or implied auto if undefined?

Reply:
The `userId` field (for the foreign key) would be the same as the `id` of the User on which it is created. It is always set by prisma for relationship fields.
In this case, it coincides with the id field.